WordPress/Sniffs/DateTime/CurrentTimeTimestampSniff.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPressCS\WordPress\Sniffs\DateTime;
+
+use WordPressCS\WordPress\AbstractFunctionParameterSniff;
+use PHP_CodeSniffer\Util\Tokens;
+
+/**
+ * Don't use current_time() to get a (timezone corrected) "timestamp".
+ *
+ * Disallow using the current_time() function to get "timestamps" as it
+ * doesn't produce a *real* timestamp, but a "WordPress timestamp", i.e.
+ * a Unix timestamp with current timezone offset, not a Unix timestamp ansich.
```

Comment:
What is `ansich`?

Reply:
https://en.wikipedia.org/wiki/Thing-in-itself ;-)